akka-cluster-sharding/src/multi-jvm/scala/akka/cluster/sharding/ClusterShardingQueriesSpec.scala

```diff
@@ -29,13 +29,20 @@ object ClusterShardingQueriesSpec {
   val shardTypeName = "DatatypeA"
 }
 
-object ClusterShardingQueriesSpecConfig
-    extends MultiNodeClusterShardingConfig(additionalConfig = """
+object ClusterShardingQueriesSpecConfig extends MultiNodeClusterShardingConfig(additionalConfig = s"""
         akka.log-dead-letters-during-shutdown = off
         akka.cluster.sharding {
           shard-region-query-timeout = 2ms
           updating-state-timeout = 2s
           waiting-for-state-timeout = 2s
+          least-shard-allocation-strategy {
+            # max 4 each time  
+            rebalance-absolute-limit = 4
+            
+            # this is a fraction of the total shard, set explicitly to 1.0 to select all shards
+            #  and force the rebalance to be based on rebalance-absolute-limit
+            rebalance-relative-limit = 1.0 
+          }
         }
```

Comment:
I don't see that the actual `GetShardRegionState` requests and asserts are retried (awaitAssert) so will this be enough?

Reply:
Good idea. That will be a better solution since more explicit to the fact that we new to wait for the shards to be rebalanced.